monitor/ui/src/main/webapp/doc/css/screen.css

```diff
@@ -0,0 +1,1279 @@
+/* Original style from softwaremaniacs.org (c) Ivan Sagalaev <Maniac@SoftwareManiacs.Org> */
```

Comment:
When you get complete "swagger-ui-master" there is a file LICENSE with the link to: http://www.apache.org/licenses/LICENSE-2.0